frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -93,17 +120,28 @@ const hydrateProjects = (state: State, dispatch: React.Dispatch<Action>) => {
       projects: string[];
     };
 
-    _.forEach(Object.keys(state[Group.PROJECTS]), p => {
-      // if the project is custom
-      if (state[Group.PROJECTS][p].custom) {
-        requestParams.projects.push(p);
-      }
-    });
+    requestParams.projects = Object.keys(state[Group.PROJECTS]);
 
     client
       .post("/v1/project/getProjects", requestParams as IClutch.project.v1.GetProjectsRequest)
       .then(resp => {
-        const { results } = resp.data as IClutch.project.v1.GetProjectsResponse;
+        const { results, partialFailures } = resp.data as IClutch.project.v1.GetProjectsResponse;
+
+        if (partialFailures && partialFailures.length) {
+          const missing: string[] = [];
+          partialFailures.forEach(failure => {
+            (failure.details || []).forEach(detail => {
+              if (_.get(detail, ["data", "disabled"])) {
```

Comment:
since implementers of the projectAPI can stub the `data` field with any value, I think this would break for users who aren't handling these kinds of errors the way we are internally https://github.com/lyft/clutch/blob/main/api/core/project/v1/project.proto#L18

Reply:
since the `HYDRATE_DEPRECATION` error case may not be relevant to all project api users, should we instead handle the partial failure error in the `HYDRATE_ERROR` action (since the partial failure response will include a message like `"x project is disabled"`) and in that action remove those projects from the state?

Reply:
Mm, good call. Guess we won't be able to check that object then.

Reply:
Removed HYDRATE_DEPRECATION and moved the logic into HYDRATE_ERROR, utilizing the message and the full details object. Might need to be even more generic